README.md

```diff
@@ -1,21 +1,25 @@
-## Auth api
+## Overview
 
-Projects are split into modules. If in future we wish to deploy separatedly (microservices and such),
-they should be split by those.
+Projects are split into modules; this would support fine-grained deployment, for example microservices. Two module 'packages' are used: `auth` and `bar`.
 
-For authentication your API can either depend on `auth-direct` project, to hit directly DB, or use
-`auth-http` for cleaner authentication via HTTP (your api does http request behind the scenes on auth-api,
-so it must be running at this point).
+**auth**
 
-There are lots of TODOs, but hopefully nothing massive.
+ * `auth-api` - what is this?
+ * `auth-core` - what is this?
+ * `auth-direct` - provides the raw authentication service, which directly access the database
+ * `auth-http` - provides an authentication service over HTTP by leveraging `auth-api`
 
-For tests you need to run docker as mentioned below.
+**bar**
+
+ * bar-api - what is this?
```

Comment:
bar api is just example api and how it is protected :) I'd like toh ave one with auth-direct and one with auth-http (so it requires auth-api to be running)